corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -412,6 +412,15 @@ hqDefine('app_manager/js/forms/case_config_ui', function () {
                 return count;
             });
 
+            self.hasDeprecatedProperties = ko.computed(function () {
+                for (const p of self.case_properties()) {
```

Comment:
where is this `case_properties` set?
Also, we can't use the deprecated dictionary we created for this?

Reply:
> where is this case_properties set?

This is getting set [here](https://github.com/dimagi/commcare-hq/blob/d57aa086c36b5631f0d1f23cbcbea10e9ce0da78/corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js#L284) in `baseMapping` which gets used by both the `caseTransaction` and `usercaseTransaction` models for the case and user property pages, respectively. These models get instantiated [here](https://github.com/dimagi/commcare-hq/blob/d57aa086c36b5631f0d1f23cbcbea10e9ce0da78/corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js#L631) and [here](https://github.com/dimagi/commcare-hq/blob/d57aa086c36b5631f0d1f23cbcbea10e9ce0da78/corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js#L709).

> we can't use the deprecated dictionary we created for this?

The deprecated dictionary contains a mapping of all the project's deprecated properties. We need to see whether any of these are being currently used to map form questions to, which is why we look through `case_properties`.

Reply:
Okay :+1: